zerver/views/registration.py

```diff
@@ -172,7 +178,13 @@ def accounts_register(
     try:
         validators.validate_email(email)
     except ValidationError:
-        return render(request, "zerver/invalid_email.html", context={"invalid_email": True})
+        # Demo org owners are not required to provide a valid email
+        # upon demo realm creation, so we can skip email validation
+        # for demo org owners.
+        if realm_creation and is_demo_organization:
+            pass
```

Comment:
I think if we're skipping validation, we should do the if statement outside the try/except block, right?  And potentially have explicit code that discards `prereg_user.email` if `is_demo_organization && realm_creation`, to ensure we never write an unvalidated email address to the database?